events/python/send_pika.py

```diff
@@ -0,0 +1,37 @@
+import time
+
+import pika
+
+from setup_pika import (
+    ARTICLE_EXCHANGE_NAME,
+    DASHBOARD_QUEUE_NAME,
+    DELIVERY_MODE_PERSISTENT,
+    get_channel
+)
+
+
+if __name__ == '__main__':
+
+    with get_channel() as channel:
+        # create an exchange of type `fanout`, will skip if already exists
+        channel.exchange_declare(exchange=ARTICLE_EXCHANGE_NAME,
+                                 exchange_type='fanout',
+                                 durable=True)
+
+        # create queue, will skip if exists
+        channel.queue_declare(queue=DASHBOARD_QUEUE_NAME, durable=True)
+
+        # bind queue to exchange, will skip if already bound
+        channel.queue_bind(exchange=ARTICLE_EXCHANGE_NAME, queue=DASHBOARD_QUEUE_NAME)
+        channel.confirm_delivery()
```

Comment:
this is about publisher confirms right?

Reply:
Yes, will add a comment to make it clear.